corehq/apps/app_manager/management/commands/migrate_adv_form_action_parent.py

```diff
@@ -0,0 +1,33 @@
+from corehq.apps.app_manager.management.commands.helpers import AppMigrationCommandBase
+from corehq.apps.app_manager.models import Application
+
+
+class Command(AppMigrationCommandBase):
+    help = "Migrate single parent index to ParentIndex list in advanced form actions."
+
+    include_builds = False  # AdvancedAction lazy-migrates reverted builds
+
+    def migrate_app(self, app_doc):
+        modules = [m for m in app_doc['modules'] if m.get('module_type', '') == 'advanced']
+        should_save = False
+        for module in modules:
+            for form in module['forms']:
+                for action_name in form.get('actions', {}):
+                    if action_name in ('load_update_cases', 'open_cases'):
+                        for action in form['actions'][action_name]:
+                            if 'parent_tag' in action:
```

Comment:
if `action['parent_tag']` is `None` or `''` I still want to remove the unused keys, and set `should_save = True` ... although I'm assuming I need to clean up `action`. Is that assumption right? Do I need to worry about that, or can I just leave `action` as-is?